fppanalysis/running_moments.py

```diff
@@ -149,3 +149,56 @@ def window_radius(cut_off_freq, time):
     samples = ((t_run_mean / dt) - 1) / 2
 
     return int(samples)
+
+def run_norm_ds(ds, radius, **kwargs):
```

Comment:
kwargs not used? If so, remove it

Reply:
I changed it now, and it is ready to merge. However, it says that there still is a conflict, even though there is only new code added?